libs/auth-api-lib/migrations/20210303105229-client-delegations-non-nullable.js

```diff
@@ -4,23 +4,13 @@ module.exports = {
   up: (queryInterface, Sequelize) => {
     return queryInterface.sequelize.query(`
     BEGIN;
-      ALTER TABLE client
-      ALTER COLUMN supports_delegation SET NOT NULL,
-      ALTER COLUMN supports_legal_guardians SET NOT NULL,
-      ALTER COLUMN supports_procuring_holders SET NOT NULL,
-      ALTER COLUMN prompt_delegations SET NOT NULL;
```

Comment:
This won't have the desired effect for applications that have already been deployed, since we use version controlled migrations, meaning they are executed automatically as a part of the deployment. This version `20210303105229` will already have been executed (at least for dev), so this migration won't change.
Rather have to create a new migration with the desired effect.

Reply:
The desired effect is to have this migration do nothing (because it fails).

Reply:
Can we resolve this then?

Reply:
@darrikonn the migration in question is not working on Staging but it was just fine on Dev. Adding a new migration is not going to help us with that. I don't like it since it will mean we have to drop the db on Dev but don't see many options. Alternative would be to "fix" the data on Staging and Prod and that is even more work. Will need to discuss how can we have avoided this situation.

Reply:
Yeah I see.. in that case, dev should be fine, since this migration won't run, and staging will just commit an empty transaction -> but sync the sequelize meta version :+1: .. so yeah, this will work. Good to get the dev vs. staging context :+1:

Reply:
Agree it shouldn't be a problem on dev.

Reply:
Not sure about the etiquette for resolving, á ég að gera það?

Reply:
Buuuut still an issue since dev and staging will have different table structure.

Reply:
Will be fixed in an upcoming PR.

Reply:
We will run this for dev:

```
npx sequelize-cli db:migrate:undo
```

Or something along those lines.

Reply:
Why is that necessary?

Reply:
@valurefugl because the migration has already been executed on Dev to completion

Reply:
But we have prepared a new migration that first sets nulls to default values, and then sets the not null constraints. That should align the environments.
Should actually have done that here, but was in too much of a rush.